templates/zoom-parts/ip-links.tpl

```diff
@@ -14,6 +14,7 @@
   <div class="btn-group">
     <a id="IPWhois-{$index}" class="btn btn-small" target="_blank" href="{$baseurl}/redir.php?tool=oq-whois&amp;data={$ipaddress}" onMouseUp="$('#IPWhois-{$index}').addClass('btn-visited');">Whois</a>
     <a id="IPWhois2-{$index}" class="btn btn-small" target="_blank" href="{$baseurl}/redir.php?tool=tl-whois&amp;data={$ipaddress}" onMouseUp="$('#IPWhois2-{$index}').addClass('btn-visited');">(alt)</a>
+    <a id="IPWhois3-{$index}" class="btn btn-small" target="_blank" href="https://www.projecthoneypot.org/ip_{$ipaddress}" onMouseUp="$('#IPWhois3-{$index}').addClass('btn-visited');">IP-Project Honeypot</a>
```

Comment:
Actually, something else I just noticed - this shouldn't be grouped with the Whois links; instead make it its own button group